deployment/helm/resource-management-policies/balloons/templates/configmap.yaml

```diff
@@ -1,10 +1,15 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: nri-resource-policy-config.default
+  name: nri-resource-policy-config
```

Comment:
Why is this being removed? 
Watcher is expecting to have .default, see here https://github.com/containers/nri-plugins/blob/66846cb397314376bc8269ce0549387c89bcc595/pkg/resmgr/agent/watcher.go#L206

Reply:
Hi, I encounter this error in the nri pod if I don't remove the .default suffix. 
```
E0727 13:32:22.818135       1 resource-manager.go:478] configuration rejected: config error: module main: no child corresponding to data nri-resource-policy.cfg
```

I don't why the error can be gone if I remove the ".default" suffix.

Reply:
Can you give a bit more context about how are you installing, which manifests you are using?

Reply:
![image](https://github.com/containers/nri-plugins/assets/12726668/3c5180bb-9069-4242-8e32-bfd3a5e2d10e)
Here is the detailed error message in the nri pod.

Also, I use the latest nri code and the helm command is 

```
sdp@b49691a74bec:~/zhi/own/nri-plugins/deployment/helm/resource-management-policies/balloons/balloons$ helm install balloon . --namespace kube-system
```

Reply:
It is strange...but I will see if I can reproduce it and dig a bit.

Reply:
I guess the problem is that we're using the same configmap in two conflicting ways:
1. mount the configmap inside the container to use for --fallback-config
2. use the same config map for dynamic configuration, as a K8s object directly watched by the daemon (and not mounted inside the container)

The only quick fix I can think of would be to have two separate configmaps, one for the fallback config and one for the dynamic configuration. Going forward we should try to get rid of the fallback config